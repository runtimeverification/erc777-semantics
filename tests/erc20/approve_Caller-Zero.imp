test.preamble();

// Testing approve(Spender,Allowance) when Spender = caller and Allowance = 0.
// Might be useful in order to throw when transferring to yourself.

test.approve(caller, 0);
