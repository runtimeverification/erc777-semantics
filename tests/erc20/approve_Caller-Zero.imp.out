7 is approving allowance 0 for 7
